python/dask_cudf/dask_cudf/backends.py

```diff
@@ -142,10 +142,12 @@ def meta_nonempty_cudf(x):
     res = cudf.DataFrame(index=idx)
     for col in x._data.names:
         dtype = str(x._data[col].dtype)
-        if dtype in ("list", "struct"):
-            # Not possible to hash and store list & struct types
-            # as they can contain different levels of nesting or
-            # fields.
+        if dtype in ("list", "struct", "category"):
```

Comment:
This fix is required because of an upstream change: https://github.com/dask/dask/pull/8898/

```python
(Pdb) ddf._meta_nonempty
   A  C  B
0  a  1  a
1  a  1  a
(Pdb) gddf._meta_nonempty
   A  C  B
0  a  a  a
1  a  a  a
```

now:

```python
(Pdb) ddf._meta_nonempty
   A  C  B
0  a  1  a
1  a  1  a
(Pdb) gddf._meta_nonempty
   A  C  B
0  a  1  a
1  a  1  a
```

Reply:
Would we want a test in upstream Dask's gpuCI to cover this breakage in the future?